Modules/getpath.c

```diff
@@ -757,10 +757,12 @@ static int
 library_to_dict(PyObject *dict, const char *key)
 {
 #ifdef MS_WINDOWS
+#ifdef Py_ENABLE_SHARED
     extern HMODULE PyWin_DLLhModule;
     if (PyWin_DLLhModule) {
         return winmodule_to_dict(dict, key, PyWin_DLLhModule);
     }
```

Comment:
Why did this access PyWin_DLLhModule when building as a static lib, but not define it in dl_nt.c. Or is it expected that users building a static lib on windows set this themselves?

Reply:
I guess it was an oversight. Building the core interpreter as a static lib precludes importing extension modules that link dynamically with "python3[x].dll". I don't think it's a tested use case on Windows.